service/storage_proxy.cc

```diff
@@ -7168,37 +7167,32 @@ void storage_proxy::cancel_write_handlers(noncopyable_function<bool(const abstra
             // iterator_guard handles safe iterator updates while allowing prompt
             // handler destruction and client response.
             cancellable_write_handlers_list::iterator_guard ig{*_cancellable_write_handlers_list, it};
-            seastar::thread::yield();
+            co_await maybe_yield();
```

Comment:
More correct is coroutine::maybe_yield()

Reply:
right, updated